libs/near/src/lib/tx.ts

```diff
@@ -51,7 +50,10 @@ export interface NearBuildTxParams {
   senderAccountID: NearAccountID;
   recipientAccountID: NearAccountID;
 }
-export interface NearSignTxOpts {}
+
+export interface NearSignTxOpts {
+  _: string;
```

Comment:
what does the `_` do? Could we comment this above?

Reply:
oh yeah, that was to get eslint to stop complaining